docs/example.md

```diff
@@ -9,7 +9,7 @@ This allows different stakeholders to understand how a project has developed.
 ## Details
 
 ```{image} _static/images/ocds_show.png
-:target: https://open-contracting.github.io/ocds-show-ppp/?load=https://raw.githubusercontent.com/open-contracting/ocds-show-ppp/master/example/full.json
+:target: https://open-contracting.github.io/ocds-show-ppp/?load=https://raw.githubusercontent.com/open-contracting-extensions/public-private-partnerships/1.0-dev/docs/examples/full_record_package.json
```

Comment:
Did you intend to change it from full to full_record_package?

Reply:
Yep. I'm not sure why it was different, since the record package is easier to navigate in the ocds show interface.